src/sentry/issues/priority.py

```diff
@@ -68,23 +71,62 @@ def get_priority_for_escalating_group(group: Group) -> PriorityLevel:
     """
     Get the priority for a group that is escalating by incrementing it one level.
     """
-
     if group.priority and group.priority == PriorityLevel.LOW:
         return PriorityLevel.MEDIUM
 
     return PriorityLevel.HIGH
 
 
+def get_priority_for_ongoing_group(group: Group) -> PriorityLevel | None:
+    if not features.has("projects:issue-priority", group.project, actor=None):
```

Comment:
Just now realizing that we'll need a separate flag for the frontend changes, since we'll need to enable this for all orgs before we start the backfill

Reply:
yeahh I was thinking about this - we can clean up flags in another PR